main.py

```diff
@@ -150,6 +150,7 @@ def main():
     PATH = get_inputs('PATH')
     COMMIT_MESSAGE = get_inputs('COMMIT_MESSAGE')
     AVATAR_SHAPE = get_inputs('AVATAR_SHAPE')
+    DEBUG_MODE = get_inputs('DEBUG_MODE')
```

Comment:
Codacy found an issue: [local variable 'DEBUG_MODE' is assigned to but never used (F841)](https://app.codacy.com/manual/bobankhshen/add-contributors/pullRequest?prid=6018022)